ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -7,6 +7,100 @@
   },
   timelineFile: 'e6s.txt',
   triggers: [
+    {
+      id: 'E6S Strike Spark',
+      regex: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexDe: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexFr: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexJa: Regexes.startsUsing({ source: ['イフリート', 'ラクタパクシャ'], id: '4BD3', capture: false }),
+      promise: function(data) {
+        let p = new Promise(async (res) => {
+          // helper function to delay the promise execution for the given time
+          const sleep = m => new Promise(r => setTimeout(r, m));
+
+          await sleep(10000);
+
+          let combatantNames = null;
+
+          // select the 4 most recent Ifrit or Raktapaksa's depending on phase
+          if (data.phase = 'ifrit') {
+              combatantNames = { names: ['Ifrit', 'イフリート', '伊弗利特', '이프리트'] };
+          } else {
+              // @TODO: Need the CN/KR names for Raktapaksa
+              combatantNames = { names: ['Raktapaksa', 'ラクタパクシャ'] };
+          }
+            
+          let combatantData = await window.callOverlayHandler({
+            call: 'getCombatants',
+            data: combatantNames
+          });
+
+          if (combatantData !== null &&
```

Comment:
style nit: could just call res() and early return here if these aren't true to avoid having to indent all of this